noxfile.py

```diff
@@ -77,6 +77,7 @@ def lint(session):
 @nox.session(python="3.8")
 def docs(session):
     session.install(*doc_dependencies)
+    session.env["PIPX__DOC_DEFAULT_PYTHON"] = "/usr/bin/python"
```

Comment:
How about instead of 
`/usr/bin/python`
we use the text
`typically the python used to execute pipx`
?

Reply:
This is a great idea, it looks much better when rendered.

I've made the change and also adjusted the line break for better fill.